content/projects/data-science-specific/natural-language-processing/_index.md

```diff
@@ -24,26 +24,41 @@ tags:
 - sentiment-analysis
 - nltk
 - textblob
-title: Natural language processing
+title: Natural language processing.
 ---
+In order to help you develop your NLP skills, we have prepared two datasets for you to work with.These datasets are designed to improve your understanding of text processing, sentiment analysis, and text classification. You have the option to work on either one dataset or both datasets. Please follow the instructions below to get started:
```

Comment:
I'm happy for learners to have a choice of datasets as long as the same skills are covered in both - do you think this is the case in the way both datasets' instructions are laid out?